api/api.go

```diff
@@ -14,7 +14,6 @@ import (
 
 type TrentoApiService interface {
 	IsWebServerUp() bool
-	GetChecksSettingsById(id string) (*webApi.JSONChecksSettings, error)
```

Comment:
Ah great! Thanks. I should have cleaned up that.
Next to this we can also remove `api/checks.go` and `api/checks_test.go`, would you mind? :innocent: